contrib/kind-helm.sh

```diff
@@ -150,7 +150,7 @@ build_ovn_image() {
     # Find all built executables, but ignore the 'windows' directory if it exists
     find ../../go-controller/_output/go/bin/ -maxdepth 1 -type f -exec cp -f {} . \;
     echo "ref: $(git rev-parse  --symbolic-full-name HEAD)  commit: $(git rev-parse  HEAD)" > git_info
-    docker build -t "${OVN_IMAGE}" -f Dockerfile.fedora .
+    docker build -t "${OVN_IMAGE}" --load -f Dockerfile.fedora .
```

Comment:
this fails on docker like this:

```
+ docker build -t ghcr.io/ovn-org/ovn-kubernetes/ovn-kube-ubuntu:helm --load -f Dockerfile.fedora .
DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
            Install the buildx component to build images with BuildKit:
            https://docs.docker.com/go/buildx/

unknown flag: --load
```

how about adding a variable that is passed as flag and set its value depending on the oci_bin?

Reply:
tks @flavio-fernandes I'll do it asap.